### ISSUE TITLE
Move buttons in the call UI to ButtonVue

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/spreed#7703]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

- [ ] Grid.vue  (3 usages found)
    - [ ] 24 <button v-if="isStripe"
    - [ ] 38 <button v-if="hasPreviousPage && gridWidth > 0"
    - [ ] 102 <button v-if="hasNextPage && gridWidth > 0"
- [ ] LocalVideo.vue  (1 usage found)
    - [ ] 58 <button v-if="isBig"
- [ ] VideoBottomBar.vue  (5 usages found)
    - [ ] 47 <button v-show="!connectionStateFailedNoRestart"
    - [ ] 60 <button v-show="!connectionStateFailedNoRestart && model.attributes.videoAvailable"
    - [ ] 71 <button v-show="!connectionStateFailedNoRestart"
    - [ ] 79 <button v-show="connectionStateFailedNoRestart"
    - [ ] 88 <button v-if="hasSelectedVideo && isBig"
- [ ] PollVotersDetails.vue  (1 usage found)
    - [ ] 24 <button slot="trigger"
            
            
Unluckily it's not too straight forward as the buttons are always on the call screen dark background and so font color, background and border color need to be hardcoded.

**Comment by nickvergessen** [comment]
This is not really trivial as there is no good matching style/type for them, especially since they are placed on videos and the semi transparent background

**Comment by nickvergessen** [comment]
So moving to 26 for now.
